jfrog-cli/artifactory/utils/docker/docker_test.go

```diff
@@ -58,7 +58,7 @@ func TestResolveRegistryFromTag(t *testing.T) {
 	}
 
 	for _, v := range imageTags {
-		result := ResolveRegistryFromTag(v.in)
+		result, _ := ResolveRegistryFromTag(v.in)
```

Comment:
I suggest we also handle the possible error returned from the *ResolveRegistryFromTag* function and fail the test with it. Actually, we can test if the we get an error in case the tag does not include a slash.

Reply:
Added a test case for error result + handling of unexpected errors